src/pickups/datastore/pickups.js

```diff
@@ -31,13 +42,15 @@ export default {
         group,
         collectors: pickup.collectorIds.map(rootGetters['users/get']),
         feedbackGivenBy: pickup.feedbackGivenBy ? pickup.feedbackGivenBy.map(rootGetters['users/get']) : [],
+        hasStarted: pickup.date <= new Date(),
```

Comment:
This is not fully reactive, you need to compare with `state.now` instead of `new Date()`.